pupil-spa/src/app/services/user/user.service.ts

```diff
@@ -16,7 +15,7 @@ export class UserService {
   data: any = {};
 
   constructor(private http: Http, private storageService: StorageService) {
-    this.loggedIn = !!this.storageService.getItem(sessionDataKey);
+    this.loggedIn = !!this.storageService.getItem(accessTokenKey);
```

Comment:
@jon-shipley bear in mind the access token is going to be HttpOnly soon.

Reply:
Yes, that's fine.  At which point we can delete this.